lesson05/assignment/index.html

```diff
@@ -1,15 +1,123 @@
-        <title>Assignment 05 - CSS Grid</title>
-    </head>
-    <body>
-        <main class="main">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <script src="https://kit.fontawesome.com/13eb2ac932.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow&family=Barlow+Condensed&family=Barlow+Semi+Condensed:wght@500&display=swap" rel="stylesheet">
+    <title>Assignment 05 - CSS Grid</title>
+  </head>
+
+  <body>
+    <a href="#main-content" class="hidden">Skip to main content</a>
+    <div class="container">
+      <header class="header">
+        <div class="header_name-logo">
+          <p>Ten Ton</p>
+          <p>Coffee</p>
+        </div>
+        <nav class="header_nav">
+          <ul class="header_nav_items">
+            <li><a href="#">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Menu</a></li>
+            <li><a href="#">Merch</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <main id="main-content">
+        <div class="main-container">
+          <p class="main-container_intro">Ten Ton Coffee offers locally roasted coffee and house-made baked goods, located on Main Street in the historic city of Waitsburg, Washington.</p>
+          <div class="main-container_articles">
+            <article>
+              <div class="main-container_articles_wrapper">
+                <img src="img/ten-ton-owners.jpeg" width="500" height="500">
+              </div>
+              <div class="main-container_articles_text">
+                <h4>Grab a warm drink and a tasty treat</h4>
+                <p>With exceptional espresso, pastries baked from scratch, gluten-free options, house-made granola, and delicious panini, Ten Ton Coffee has what you crave</p>
+              </div>
+            </article>
+            <article>
+              <div class="main-container_articles_wrapper">
+                <img src="img/ten-ton-mask.jpg" width="500" height="500">
+              </div>
+              <div class="main-container_articles_text">
+                <h4>We're open!</h4>
+                <p>Our fabulous take-out window is open daily, 7:30am - 2pm. Come grab a coffee and say hello!</p>
+              </div>
+            </article>
+          </div>
           
-        </main>
-    </body>
+          <section>
+            <h2>Instagram</h2> <!-- To be API feed -->
+            <div class="main-container_instagram">
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-1.jpg" alt="Instagram image 1" width="250" height="250">
+              </div>
```

Comment:
Since all these div-wrapped images have the same class, I don't see a need to have a class on them at all. The class assigned to their parent on line 59 can be used to select them: `.main-container_instagram > div`